CONTRIBUTING-docs.md

```diff
@@ -64,6 +64,24 @@ After you have been assigned to an issue, fork the [json-schema-org/website](htt
 
 To propose modifications to our documentation that do not have an issue in the [documentation board](https://github.com/orgs/json-schema-org/projects/16), you can [create a GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-repository) and use the *Documentation* template to describe the changes. After that, you can decide whether you want to work on the changes yourself or let someone else claim the issue. If you decide to work on the issue, assign it to yourself and commit the changes to a new branch. 
 
+## Add Metadata to Newly Created Markdown Files
+When contributing to the documentation, it's important to include metadata in every newly created markdown file. This metadata helps organize content and ensure consistency across all pages in the documentation.When you create a new markdown file, add metadata to the file. The metadata should include the title of the page, the author, and the date of the last update. Here is an example of metadata:
```

Comment:
There is a whitespace error between the first and second sentences. "_documentation.when_".